src/blockscout/blockscout_launcher.star

```diff
@@ -0,0 +1,131 @@
+shared_utils = import_module("../shared_utils/shared_utils.star")
+constants = import_module("../package_io/constants.star")
+postgres = import_module("github.com/kurtosis-tech/postgres-package/main.star")
+
+IMAGE_NAME_BLOCKSCOUT = "blockscout/blockscout:6.0.0"
+IMAGE_NAME_BLOCKSCOUT_VERIF = "ghcr.io/blockscout/smart-contract-verifier:v1.6.0"
+
+SERVICE_NAME_BLOCKSCOUT = "blockscout"
+
+HTTP_PORT_ID = "http"
+HTTP_PORT_NUMBER = 4000
+HTTP_PORT_NUMBER_VERIF = 8050
+
+BLOCKSCOUT_MIN_CPU = 100
+BLOCKSCOUT_MAX_CPU = 1000
+BLOCKSCOUT_MIN_MEMORY = 1024
+BLOCKSCOUT_MAX_MEMORY = 2048
+
+BLOCKSCOUT_VERIF_MIN_CPU = 10
+BLOCKSCOUT_VERIF_MAX_CPU = 1000
+BLOCKSCOUT_VERIF_MIN_MEMORY = 10
+BLOCKSCOUT_VERIF_MAX_MEMORY = 1024
+
+USED_PORTS = {
+    HTTP_PORT_ID: shared_utils.new_port_spec(
+        HTTP_PORT_NUMBER,
+        shared_utils.TCP_PROTOCOL,
+        shared_utils.HTTP_APPLICATION_PROTOCOL,
+    )
+}
+
+VERIF_USED_PORTS = {
+    HTTP_PORT_ID: shared_utils.new_port_spec(
+        HTTP_PORT_NUMBER_VERIF,
+        shared_utils.TCP_PROTOCOL,
+        shared_utils.HTTP_APPLICATION_PROTOCOL,
+    )
+}
+
+
+def launch_blockscout(plan, el_client_contexts, persistent):
+    postgres_output = postgres.run(
+        plan,
+        service_name="{}-postgres".format(SERVICE_NAME_BLOCKSCOUT),
+        database="blockscout",
+        extra_configs=["max_connections=1000"],
+        persistent=persistent,
```

Comment:
This won't enable persistence the way you expect it. 

It needs to be defined like this: 

https://github.com/kurtosis-tech/ethereum-package/blob/main/src/el/geth/geth_launcher.star#L333

So its the files parameter that you want to enable persistence on. And mount the pvc to whatever datadir you want to keep persistence of.